changelog/fragments/scorecard-custom-image-relocate.yaml

```diff
@@ -0,0 +1,14 @@
+entries:
+  - description: Updated custom scorecard test example
+    kind: change
+    breaking: false
+    migration:
+      header: Update custom scorecard test example
+      body: |
+        The custom-scorecard-tests image is built like the other
+        scorecard images with this change, the Makefile is updated
+        to produce the custom-scorecard-tests image.  This makes it
+        easier to build the image using the normal build process and
+        will make testing the custom image easier, which will be
+        addressed in another PR.
```

Comment:
Would this affect the users? if not, I do not think that we need the fragment.

Reply:
this PR doesn't impact users yet, another forthcoming PR however will make a change to the website docs to update some links to the new relocated file locations, so I'll add the changelog into that PR.